src/hn-topstories/HnTopstories.js

```diff
@@ -1,8 +1,15 @@
 import { LitElement, html } from "lit";
 import { ScopedRegistryHost } from "@lit-labs/scoped-registry-mixin";
 import { topStories } from "../api/topStories.js";
+import { HnStory } from "../hn-story/HnStory.js";
 
 export class HnTopstories extends ScopedRegistryHost(LitElement) {
+  static get elementDefinitions() {
+    return {
+      "hn-story": HnStory,
```

Comment:
If you preer to do this definition dinamically let me know and I could do the change to define the component con the `initial` state

Reply:
I don't know the benefits of each one 🤷

Reply:
In this case the static import is ok, as you are going to use always the HnStory component. However, if at some point the type of component could be different, then dynamic import would be useful.